apps/game_backend/lib/game_backend/curse_of_mirra/quests.ex

```diff
@@ -65,6 +70,43 @@ defmodule GameBackend.CurseOfMirra.Quests do
     Quest.changeset(quest, attrs)
   end
 
+  def get_daily_quest(daily_quest_id) do
+    q = from(dq in DailyQuest, preload: [:quest], where: dq.id == ^daily_quest_id)
+
+    Repo.one(q)
+  end
+
+  def get_users_daily_quests(user_id) do
+    q = from(dq in DailyQuest, preload: [:quest], where: dq.user_id == ^user_id)
+
+    Repo.all(q)
+  end
+
+  @doc """
+  Get all %DailyQuest{} for the user_id that were inserted today and were rerolled
```

Comment:
I think this is a better doc.
```suggestion
  Receives a user id and a daily quest type.
  Returns a list of DailyQuest for the given user and type that were inserted today an their status are :rerolled.
```